test/Testing.Tests/Rocket.Surgery.Extensions.Testing.Tests.csproj

```diff
@@ -15,9 +15,4 @@
         <Content CopyToOutputDirectory="PreserveNewest" Include="Fixtures/**/*.csproj;Fixtures/**/*.props;Fixtures/**/*.targets" />
         <Compile Remove="Fixtures/**/*.cs" />
     </ItemGroup>
-    <ItemGroup>
```

Comment:
@RLittlesII this old school reference was causing the jetbrains cleanupcode to remove the using that contained the extension methods fucking wild man.